modules/core/src/main/java/org/apache/fluo/core/oracle/Stamp.java

```diff
@@ -15,11 +15,18 @@
 
 package org.apache.fluo.core.oracle;
 
+import org.apache.fluo.accumulo.util.ColumnConstants;
+import com.google.common.base.Preconditions;
+
 public class Stamp {
   private final long txStamp;
   private final long gcStamp;
 
   Stamp(long stamp, long gcStamp) {
+	long b= stamp & ColumnConstants.PREFIX_MASK ;
+	Preconditions.checkArgument(b==0?true:false, "should be zero");
```

Comment:
I think  `b==0?true:false` can be replaced with `b==0`.  It would be nice to include more info in the error message, somethingl ike `timestamp prefix should be zero`